src/docs/reference/modules/OpenId/README.md

```diff
@@ -2,32 +2,42 @@
 
 ## OpenID Connect Module
 
-`OrchardCore.OpenId` provides the following features:
+`OrchardCore.OpenId` provides robust OpenID Connect functionality, enabling Orchard Core to act as an OpenID Connect server and client. The following features are available:
 
-- Core Components
-- Authorization Server
-- Management Interface
-- Token Validation
-- OIDC Client
+- **Core OpenID Connect Services**
+- **OpenID Connect Authorization Server**
+- **OpenID Connect Management UI**
+- **OpenID Connect Authorization Server**
+- **OpenID Connect Client Integration**
 
-## Core Components
+## Core OpenID Connect Services
 
-Registers the core components used by the OpenID module.
+This feature provides the essential services that underpin all other OpenID Connect features within Orchard Core. It includes fundamental components needed for secure communication, token handling, and user authentication.
 
-## Management Interface
+## OpenID Connect Management UI
 
-Allows adding, editing and removing the registered applications.
+Adds a management interface to the Orchard Core admin dashboard, enabling users to manage OpenID Connect applications, define and modify scopes, and configure application permissions through a user-friendly UI.
 
-## Authorization Server
+## OpenID Connect Authorization Server
 
-Enables authentication of external applications using the OpenID Connect/OAuth 2.0 standards.  
-It is based on the [`OpenIddict`](https://github.com/openiddict/openiddict-core) library allowing Orchard Core to act as identity provider to support token authentication without the need of an external identity provider.
+Allows Orchard Core to function as an OpenID Connect authorization server, also referred to as an identity provider (IdP). This feature enables authentication and the issuance of tokens, conforming to OpenID Connect and OAuth 2.0 standards.
 
-- Orchard Core can also be used as an identity provider for centralizing the user access permissions to external applications.
-- Orchard Core services.
-  - The authorization server feature maintains its own private JWT/validation handler instance for the userinfo API endpoint. This way, you don't have to enable the token validation feature for current tenant.
+Key points:
+- Orchard Core can serve as a centralized identity provider, allowing external applications to authenticate users and manage access control.
+- Powered by the [`OpenIddict`](https://github.com/openiddict/openiddict-core) library, this feature supports token-based authentication without requiring an external identity provider.
+- The Authorization Server maintains a private JWT/validation handler for the `/connect/userinfo` API endpoint, so you don't need to enable the Token Validation feature for the current tenant.
```

Comment:
Not specific to your PR since that text was already there but maybe something like that should be clearer/less specific?

```suggestion
- The authorization server takes care of validating the access tokens received by the `/connect/userinfo` API endpoint, so you don't need to enable the token validation feature for the current tenant.
```